docs/specification.md

```diff
@@ -882,3 +884,8 @@ Motivation:
 
 The deployment of contract with metadata has an extra step: a dedicated contract for carrying metadata has to be originated first.
 Then the baseDAO contract should include the reference to a metadata key in the contract in order to be compliant with TZIP-016.
+
+## Delployment with metadata that is on IPFS
```

Comment:
I think that this could be merged with the previous section, after all it's either way about deploying with metadata and the second step (include the reference) is the same in both cases (except for the different URI).

Reply:
Make sense, I've merged them now.